javascript/javascript2/week3/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <button id="click-button">Click</button> <br>
+    <button id="location"> Log location</button><br>
+    <p id="yourLocation"></p>
+     
+
```

Comment:
do not leave empty spaces without reason
use formatting plugins to format the code

Reply:
Sure, I will remember that next time :)